packages/terra-avatar/src/common/AvatarUtils.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import TerraImage from 'terra-image';
+import styles from './Avatar.module.scss';
+
+const cx = classNames.bind(styles);
+
+const AVATAR_VARIANTS = {
+  USER: 'user',
+  FACILITY: 'facility',
+  SHARED_USER: 'shared-user',
+};
+
+const COLOR_VARIANTS = [
+  'one',
+  'two',
+  'three',
+  'four',
+  'five',
+  'six',
+  'seven',
+  'eight',
+  'nine',
+  'ten',
+];
+
+/**
+ * Generates a hash, based on a given string s.
+ * https://werxltd.com/wp/2010/05/13/javascript-implementation-of-javas-string-hashcode-method/
+ * @param {*} s
+ */
+const calculateHash = (s) => {
+  let hash = 0;
+  let char;
+
+  if (s.trim().length === 0) {
+    return hash;
+  }
+
+  /* eslint-disable no-bitwise */
+  for (let i = 0; i < s.length; i += 1) {
+    char = s.charCodeAt(i);
+    hash = (hash << 5) - hash + char;
+    hash &= hash; // Convert to 32bit integer
+  }
+  /* eslint-enable no-bitwise */
+
+  return Math.abs(hash); // Always return positive hash
+};
+
+/**
+ *  Returns a color variant based on a given hashValue.
+ * @param {*} hashValue
```

Comment:
Do you want to put some parameter types/descriptions in these doc blocks? They're not super useful otherwise, and I wouldn't be opposed to just removing them outright.

Reply:
good call, removed https://github.com/cerner/terra-core/pull/2061/commits/90bc73e84c1d1bf39b580c074b951bcc5293b5e5

some extension or setting auto-magically generates them. i should probably figure that out haha